MSAL/src/MSIDInteractiveRequestParameters+MSALRequest.m

```diff
@@ -37,12 +37,18 @@ - (BOOL)fillWithWebViewParameters:(MSALWebviewParameters *)webParameters
                     customWebView:(WKWebView *)customWebView
```

Comment:
Just to confirm this will be moved to CommonCore right?

Reply:
as discussed, yes.